### DISCUSSION TITLE
Possible to get the length/size of a DB from the PLC?

### DISCUSSION THREAD

**Discussion by mattbue** [gijzelaerr/python-snap7#536] [Q&A] [answered]
Hi,
by calling the method `list_blocks_of_type` I can get the list of DBs which are on the plc.
After that is it possible to get the length/size (no of bytes) of a specifc DB?

Thanks!

**Comment by gijzelaerr** [comment, marked as answer]
Closing as stale. Yes, you can use `get_block_info(block_type, block_number)` which returns block metadata including size information (`mc7_size` field).
